components/style/matching.rs

```diff
@@ -313,7 +316,7 @@ trait PrivateMatchMethods: TElement {
         // in addition to the unvisited styles.
 
         let mut tasks = UpdateAnimationsTasks::empty();
-        if self.needs_animations_update(context, old_values.as_ref(), new_values) {
+        if self.needs_animations_update(context, old_values.as_ref().map(|s| &**s), new_values) {
```

Comment:
Do you really need the asterisks here? Or even the `&`? `s` is a `&_` and the method isn't generic, right?

Reply:
I'll check, I just grabbed from a few lines above.

Reply:
Err, below

Reply:
Happens to be needed, because it's in a `map`... shrug